core/chain-configs/src/test_utils.rs

```diff
@@ -218,3 +222,19 @@ pub fn get_initial_supply(records: &[StateRecord]) -> Balance {
     }
     total_supply
 }
+
+pub fn test_cloud_archival_configs(
+    cloud_archival_dir: PathBuf,
```

Comment:
nit: you can use `impl Into<PathBuf>` if you want to avoid the `into()` on literals